packages/common/src/store/pages/collection/types.ts

```diff
@@ -9,8 +7,9 @@ import {
   Status,
   LineupTrack
 } from 'models'
+import type { Dayjs } from 'utils/dayjs'
 
-export type CollectionTrack = LineupTrack & { dateAdded: Moment } & {
+export type CollectionTrack = LineupTrack & { dateAdded: Dayjs } & {
```

Comment:
I don't wonder if we shouldn't alias this to an internal date type and then just set that to Dayjs?
It's always felt a little strange to me that we specifically require a Moment or Dayjs object instead of taking a standard date or date string and using the library in places where we need to do date math or conversion.

Reply:
I agree... but maybe won't change it here just yet. follow up PR!